aspnetcore/mvc/models/file-uploads.md

```diff
@@ -692,24 +692,32 @@ Other Kestrel limits may apply for apps hosted by Kestrel:
 * [Maximum client connections](xref:fundamentals/servers/kestrel#maximum-client-connections)
 * [Request and response data rates](xref:fundamentals/servers/kestrel#minimum-request-body-data-rate)
 
-### IIS content length limit
+### IIS content length and request body size limits
 
-The default request limit (`maxAllowedContentLength`) is 30,000,000 bytes, which is approximately 28.6MB. Customize the limit in the *web.config* file:
+The default request limit (`maxAllowedContentLength`) is 30,000,000 bytes, which is approximately 28.6 MB. Customize the limit in the `web.config` file. In the following example, the limit is set to 50 MB (52,428,800 bytes):
 
 ```xml
 <system.webServer>
   <security>
     <requestFiltering>
-      <!-- Handle requests up to 50 MB -->
       <requestLimits maxAllowedContentLength="52428800" />
     </requestFiltering>
   </security>
 </system.webServer>
 ```
 
-This setting only applies to IIS. The behavior doesn't occur by default when hosting on Kestrel. For more information, see [Request Limits \<requestLimits>](/iis/configuration/system.webServer/security/requestFiltering/requestLimits/).
+The `maxAllowedContentLength` setting only applies to IIS. For more information, see [Request Limits `<requestLimits>`](/iis/configuration/system.webServer/security/requestFiltering/requestLimits/).
 
-Limitations in the ASP.NET Core Module or presence of the IIS Request Filtering Module may limit uploads to either 2 or 4 GB. For more information, see [Unable to upload file greater than 2GB in size (dotnet/AspNetCore #2711)](https://github.com/dotnet/AspNetCore/issues/2711).
+Increase the maximum request body size for the HTTP request by setting <xref:Microsoft.AspNetCore.Builder.IISServerOptions.MaxRequestBodySize%2A?displayProperty=nameWithType> in `Startup.ConfigureServices`. In the following example, the limit is set to 50 MB (52,428,800 bytes):
```

Comment:
You shouldn't need to do this now.

Reply:
Ok ... I'll split that bit out but keep the other nit updates here.

Reply:
... but your update ... it will be for which release? I think we would need to keep this based on the version of the doc, right?

Reply:
Ah ... 5.0. Ok ...... I'll version that for <5.0.